src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataTypes.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Button } from "@madie/madie-design-system/dist/react";
+import AddCircleOutlineIcon from "@mui/icons-material/AddCircleOutline";
+
+export default function DataTypes({ selectedDataElements }) {
+  const capitalizeFirstLetter = (w) => w.charAt(0).toUpperCase() + w.slice(1);
```

Comment:
looks like this is lodash `_.capitalize` https://lodash.com/docs/4.17.15#capitalize